cmd/gokr-stat/stat.go

```diff
@@ -85,21 +87,42 @@ func printStats() error {
 	type processAndFormatter interface {
 		ProcessAndFormat(map[string][]byte) []stat.Col
 	}
-	modules := []processAndFormatter{
-		&cpu.Stats{},
-		&disk.Stats{},
-		&sys.Stats{},
-		&net.Stats{},
-		&mem.Stats{},
+
+	var modules []processAndFormatter
+
+	if *thermalFlag {
```

Comment:
We should probably have a modulesFromFlag(*enabledModules) function here to reduce duplication with webstat

Reply:
I was thinking something like this 
```go
type processAndFormatter interface {
		ProcessAndFormat(map[string][]byte) []stat.Col
}

func modulesFromFlag(enabledModules string) []processAndFormatter {
	if strings.Contains(enabledModules, "thermal") {
		return []processAndFormatter{
			&cpu.Stats{},
			&disk.Stats{},
			&sys.Stats{},
			&net.Stats{},
			&mem.Stats{},
			&thermal.Stats{},
		}
	} else {
		return []processAndFormatter{
			&cpu.Stats{},
			&disk.Stats{},
			&sys.Stats{},
			&net.Stats{},
			&mem.Stats{},
		}
	}
}
```
but I am not sure what file I should put this function. what do you think @stapelberg ?

Reply:
In my opinion that function should reside on `/stat/stat.go` but since we are using `cpu`, `disk`, etc we have to import that to the `/stat/stat.go` are you fine with that? @stapelberg

Reply:
I don’t think that’s possible because it will cause an import cycle.

Let’s instead put this function into a new package, maybe `internal/statflag` for now.